numpy/_core/tests/test_simd.py

```diff
@@ -160,7 +160,7 @@ def test_operators_logical(self):
         assert vor == data_or
 
         data_xor = [a ^ b for a, b in zip(data_a, data_b)]
-        vxor = getattr(self, "xor")(vdata_a, vdata_b)
+        vxor = self.xor(vdata_a, vdata_b)
         assert vxor == data_xor
 
         vnot = getattr(self, "not")(vdata_a)
```

Comment:
I wonder why this was not changed.

Reply:
I guess this is an unusual construct that is currently not caught by ruff.

Reply:
I think it is because one cannot do `class A: not = bla` - it is a reserved keyword (unlike `xor`).